Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "course-buddies"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+env_logger = "0.9.0"
+futures = "0.3.21"
+hyper = "0.14.18"
```

Comment:
these are both pretty low level, i'd probably use actix for server, and it will come with futures/tokio included